<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-63Q7D3R" failures="1" tests="4" name="day08.TestngListener_demo1" time="1.122" errors="1" timestamp="02 Aug 2024 12:11:03 GMT" skipped="1">
  <testcase classname="day08.TestngListener_demo1" name="testmethod1" time="0.064"/>
  <testcase classname="day08.TestngListener_demo1" name="testmethod3" time="1.046">
    <error message="Method day08.TestngListener_demo1.testmethod3() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method day08.TestngListener_demo1.testmethod3() didn't finish within the time-out 1000
at java.base/sun.invoke.util.Wrapper.isSubwordOrInt(Wrapper.java:127)
at java.base/java.lang.invoke.MethodTypeForm.<init>(MethodTypeForm.java:170)
at java.base/java.lang.invoke.MethodTypeForm.findForm(MethodTypeForm.java:223)
at java.base/java.lang.invoke.MethodType.makeImpl(MethodType.java:407)
at java.base/java.lang.invoke.MethodType.methodType(MethodType.java:373)
at java.base/java.lang.invoke.MethodTypeForm.canonicalize(MethodTypeForm.java:253)
at java.base/java.lang.invoke.MethodTypeForm.findForm(MethodTypeForm.java:220)
at java.base/java.lang.invoke.MethodType.makeImpl(MethodType.java:407)
at java.base/java.lang.invoke.MethodType.insertParameterTypes(MethodType.java:499)
at java.base/java.lang.invoke.VarHandle$AccessDescriptor.<init>(VarHandle.java:2005)
at java.base/java.lang.invoke.MethodHandleNatives.varHandleOperationLinkerMethod(MethodHandleNatives.java:553)
at java.base/java.lang.invoke.MethodHandleNatives.linkMethodImpl(MethodHandleNatives.java:497)
at java.base/java.lang.invoke.MethodHandleNatives.linkMethod(MethodHandleNatives.java:485)
at java.base/java.util.concurrent.FutureTask.setException(FutureTask.java:300)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:322)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- testmethod3 -->
  <testcase classname="day08.TestngListener_demo1" name="testmethod2" time="0.012">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at day08.TestngListener_demo1.testmethod2(TestngListener_demo1.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testmethod2 -->
  <testcase classname="day08.TestngListener_demo1" name="testmethod4" time="0.000">
    <skipped/>
  </testcase> <!-- testmethod4 -->
</testsuite> <!-- day08.TestngListener_demo1 -->
